borg/remote.py

```diff
@@ -414,6 +414,9 @@ class RepositoryCache(RepositoryNoCache):
 
     Caches Repository GET operations using a local temporary Repository.
     """
+    # maximum object size that will be cached, 64 kiB.
+    THRESHOLD = 2**16
```

Comment:
Yes. Bandwidth-wise the small chunks will have the largest impact when cached (uplink latency × bandwidth ...). I think it's a sufficient workaround for most cases (and the borg help does already say that it may need lots of space ...

> where temporary files are stored (might need a lot of temporary space for some operations)

Reimplemented cache goes into master.